_sass/_layout.scss

```diff
@@ -162,7 +162,7 @@ $nav-image: $baseurl + "/images/menu.svg";
 	background-color: $content-background-color;
 	color: $content-color;
 
-	h3::before {
+	h3:not(:first-child):before {
```

Comment:
We seem to have a lot of articles starting with an `H3` that end up with a big margin and border -- this prevents that 👍